app/config/settings_base.py

```diff
@@ -121,6 +123,7 @@
 # https://docs.djangoproject.com/en/5.0/howto/static-files/
 
 STATIC_URL = 'static/'
+STATIC_ROOT = BASE_DIR / 'var' / 'www' / 'service_control' / 'static_files'
```

Comment:
This string seems a bit weird

Reply:
Totally. I took this from [settings_prod.py in service-stac](https://github.com/geoadmin/service-stac/blob/674b1ab1212a79dd7b8463df642300ec03b535cf/app/config/settings_prod.py#L180). What would be a better path?

Reply:
`BASE_DIR` is a `Path` object from `pathlib`, not a raw string. `pathlib` supports or even suggests to construct paths like this. So I think it's ok to leave it like this. If you want you can add a comment to make this clear.